charts/hcp-terraform-operator/templates/clusterrole.yaml

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: MPL-2.0
 
```

Comment:
Please add a three-dash delimiter (`---`) between the copyright headers and the actual YAML document. Otherwise, the line following the `if` condition is removed when `rbac.create` is set to `true` due to the whitespace control `-}}` character.

_This applies to all changes in the RBAC files, but I will only leave one comment._

Reply:
done